contracts/utils/cryptography/draft-ERC7739Signer.sol

```diff
@@ -108,9 +115,16 @@ abstract contract ERC7739Signer is EIP712, IERC1271 {
     /**
      * @dev Signature validation algorithm.
      *
+     * To ensure there's no way to inherit from this ERC-7739 signer and still be able to sign raw messages,
+     * this function provides a nested EIP-712 hash. Developers must implement only this function to ensure
+     * no raw message signing is possible.
+     *
      * WARNING: Implementing a signature validation algorithm is a security-sensitive operation as it involves
      * cryptographic verification. It is important to review and test thoroughly before deployment. Consider
      * using one of the signature verification libraries ({ECDSA}, {P256} or {RSA}).
      */
-    function _validateSignature(bytes32 hash, bytes calldata signature) internal view virtual returns (bool);
+    function _validateNestedEIP712Signature(
```

Comment:
What is the motivation for this change of name. I personnally prefer the simplicity and clarity of `_validateSignature`, and I don't see why we need to expose details like "NestedEIP712" in the naming.

Reply:
You're right, I'm reverting this change. I intended to convey that the hash passed into this function is already "safe" to clarify why the AccountBase has an `_userOpSignedHash` function.

We can rediscuss that afterwards